routing/routing.go

```diff
@@ -268,6 +269,7 @@ type Routing struct {
 	firstLoad         chan struct{}
 	firstLoadSignaled bool
 	quit              chan struct{}
+	Metrics           metrics.Metrics
```

Comment:
Please make field unexported like others

Reply:
Which way else, via param?

Reply:
Ah, yeah, we would need to add it to the routing.Options

Reply:
applied!